src/components/filter-selector/FilterSelector.tsx

```diff
@@ -141,17 +140,17 @@ const FilterSelector = <T extends Pick<CategoryNameInterface, '_id'>>({
     : t('cooperationsPage.tabs.all')
 
   return (
-    <Box id='menu-filter' sx={spliceSx(styles.root, customSx?.root)}>
-      <Typography sx={styles.text}>{title}:</Typography>
-      <Typography sx={styles.chosenFilters}>{chosenFiltersText}</Typography>
-      <IconButton
-        disableRipple
+    <>
+      <AppButton
+        id='menu-filter'
```

Comment:
What is "id" prop used for here?

Reply:
Maybe this can be good use case for useId hook, because if we try to reuse this component in other places something can go wrong because id will not be unique

Reply:
Thank you. Replaced id with ref, deleted document.getById